src/main.rs

```diff
@@ -36,27 +31,13 @@ fn panic(_info: &core::panic::PanicInfo) -> ! {
 
 #[entry]
 fn main() -> ! {
-    let peripherals = Peripherals::take();
-    let system = peripherals.SYSTEM.split();
-    let clocks = ClockControl::max(system.clock_control).freeze();
+    let peripherals = hal::init(hal::Config::default());
```

Comment:
Just a note: not sure if there was any reason to use clocks at max speed, but we aren't using them now

Reply:
Not sure if it matters either but will update it just to keep it consistent with what we had, thanks

Reply:
And the stub just times out if I try to use UART. Will test with some other chips in a bit and see if there's any difference.